project-addons/crm_claim_rma_custom/crm_claim.py

```diff
@@ -34,10 +34,22 @@ class equivalent_products_wizard(models.TransientModel):
 
 
 class CrmClaimRma(models.Model):
-
     _inherit = "crm.claim"
     _order = "id desc"
 
+    @api.one
+    def _has_category(self, claim):
+        has_category = False
+        for category in claim.category_id:
+            if category.parent_id.id == self.env.ref('__export__.res_partner_category_28').id:
```

Comment:
Ok, pero tendrás que usar un id externo nuevo y os creará un tag nuevo, que luego tendréis que actualizar para que substituya al 28 como padre de otros tags y el 28 borrarlo.